### DISCUSSION TITLE
how to build and host static docs ?

### DISCUSSION THREAD

**Discussion by agordon** [twbs/bootstrap#36612] [General]
Hello,

I'd like to host a local copy of the bootstrap documentation - preferably static files (i.e. without `npm run docs-serve`).
I was successful in locally running `npm run docs-serve` (and view them by visiting `http://localhost:9001`) - but I wasn't able to use the html files from `_site` directly as static files. It seems most of the HTMLs contain a tiny `<head>` section that redirects back to `https://getbootstrap.com/....` .

What am I missing? is there a different method ?

Thanks!

**Comment by coliff** [comment]
Run `npm run docs-build` to build the docs to the `_site` dir. If you want to host them elsewhere you can change the canonical URL of the site in the `baseURL` in the `config.yml` file. Hope that helps!
